experimental/FTheoryTools/src/auxiliary.jl

```diff
@@ -179,8 +179,8 @@ function _kodaira_type(id::MPolyIdeal{<:MPolyRingElem}, ords::Tuple{Int64, Int64
 
     # Create new ring with auxiliary variable to construct the monodromy polynomial
     R = parent(f)
-    S, (_psi, ) = polynomial_ring(QQ, ["_psi"; [string(v) for v in gens(R)]], cached = false)
-    ring_map = hom(R, S, gens(S)[2:end])
+    S, (_psi, _old_gens) = polynomial_ring(QQ, [:_psi; symbols(R)]; cached = false)
+    ring_map = hom(R, S, _old_gens)
```

Comment:
Does this actually work? Don't we need
```suggestion
    S, (_psi, _old_gens...) = polynomial_ring(QQ, [:_psi; symbols(R)]; cached = false)
    ring_map = hom(R, S, _old_gens)
```
or maybe nicer:
```suggestion
    S, (_psi, _old_gens) = polynomial_ring(QQ, [:_psi], symbols(R); cached = false)
    ring_map = hom(R, S, _old_gens)
```

Reply:
good catch. thanks!